examples/controlplane/kustomization.yaml

```diff
@@ -5,3 +5,5 @@ resources:
 - controlplane.yaml
 configurations:
 - kustomizeconfig.yaml
+patchesStrategicMerge:
+- kustomizeversions.yaml
```

Comment:
I worry a bit about applying this to the default examples. This seems like something we should apply *only* in the case of the conformance of kubernetes tip rather than always apply.

Reply:
I agree. i don't know customize enough. is there a better way? 

worst case i can append those two lines and then run kustomize (when needed)?

Reply:
+1 to appending as needed in the short term.

Reply:
fwiw, the shell script is a no-op when CI_VERSION is not set.

Reply:
I mainly worry that this will affect people that are currently consuming these files.